tutorials/tts/FastPitch_MixerTTS_Training.ipynb

```diff
@@ -183,32 +165,23 @@
   },
   {
    "cell_type": "markdown",
-   "id": "54ec3c5e",
-   "metadata": {},
    "source": [
     "# Preprocessing"
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
-   "id": "7ef87e31",
-   "metadata": {},
    "outputs": [],
    "source": [
+    "from nemo_text_processing.g2p.modules import EnglishG2p\n",
     "from nemo.collections.tts.torch.data import TTSDataset\n",
     "from nemo_text_processing.text_normalization.normalize import Normalizer\n",
-    "from nemo_text_processing.g2p.modules import EnglishG2p\n",
-    "from nemo.collections.common.tokenizers.text_to_speech.tts_tokenizers import (\n",
-    "    EnglishPhonemesTokenizer,\n",
-    "    EnglishCharsTokenizer,\n",
-    ")"
+    "from nemo.collections.tts.torch.tts_tokenizers import EnglishPhonemesTokenizer, EnglishCharsTokenizer"
```

Comment:
I still see the import here as coming from the TTS collection, could you revert this back to the path in `main`?

Reply:
yes. Fixed already, forgot to push.